libs/domain/picking/src/components/filters/filters.styles.ts

```diff
@@ -0,0 +1,21 @@
+import { css } from 'lit';
+
+export const styles = css`
+  oryx-heading {
+    flex-grow: 1;
+    text-align: center;
+    font-weight: 600;
+  }
+
+  oryx-modal::part(close) {
+    --oryx-icon-size: 24px;
+    --_margin: 0;
+
+    padding: 0;
+    max-height: 26px;
+  }
```

Comment:
can we make this small window driven? Meaning, icon button will have increased size on small screen? Or modal will show increased icon button small screen?